.trunk/trunk.yaml

```diff
@@ -0,0 +1,49 @@
+# This file controls the behavior of Trunk: https://docs.trunk.io/cli
+# To learn more about the format of this file, see https://docs.trunk.io/reference/trunk-yaml
+version: 0.1
+cli:
+  version: 1.22.8-beta.4
```

Comment:
[latest version to fix ci issues](https://trunkcommunity.slack.com/archives/C0205B6KK8X/p1730397115419999?thread_ts=1730071625.793899&cid=C0205B6KK8X)